lang/cs/Org.Apache.REEF.IO/Org.Apache.REEF.IO.csproj

```diff
@@ -88,8 +88,8 @@ under the License.
     <Reference Include="System.Core" />
     <Reference Include="System.Management" />
     <Reference Include="System.Net.Http" />
-    <Reference Include="System.Spatial, Version=5.6.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>$(PackagesDir)\System.Spatial.5.6.4\lib\net40\System.Spatial.dll</HintPath>
+    <Reference Include="System.Spatial, Version=5.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Path is different here as well.

Reply:
These paths were auto generated by Visual Studio. Upon investigation, it appears fairly common for minor version numbers to not match.

Reply:
ok disregard my comment then, thanks for looking.